Automattic/wp-calypso
# Changed Dockerfile to use "npm run build" instead of "make build"

https://github.com/Automattic/wp-calypso/pull/13536 has been merged for a few days already, and things haven't exploded. It's time for "part 2" on the war against Makefile.

https://github.com/Automattic/wp-calypso/pull/13536 added an entire build system composed of `npm` scripts, but that system wasn't used in production builds (to lower the risk of that change). This PR changes the `Dockerfile` file to use `npm run build` instead of `make run`.

**To test:**
* Run `cd public_html && php bin/calypso-secrets .config/secrets.php` in your sandbox, and paste the output of that command into `config/secrets.json` in your Calypso directory.
* Add `127.0.0.1       wpcalypso.wordpress.com` to your `/etc/hosts` file. You'll also need to be unproxied, and probably flush the DNS cache or use a different browser. If you have any process using port `80`, stop it.
* Run `npm run docker-build`. It should take a while.
* Run `npm run docker`. As soon as you see `wp-calypso booted in XXXms`, you can open `http://wpcalypso.wordpress.com` and you should see Calypso working as expected.
